jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -20,7 +22,7 @@ toc::[]
 ==== Overview 
 Output processor is responsible for handling the output from Nomad clients. It is a key service to scale if you find your system slowing down. We recommend increasing the output processor replica set to scale the service up to meet demand. 
 
-Access the KOTS admin console. Get to the KOTS admin console by running the following, substituting your namespace: `kubectl kots admin-console -n <namespace>`
+Access the KOTS admin console. Get to the KOTS admin console by running the following, substituting your namespace: `kubectl kots admin-console -n <YOUR_CIRCLECI_NAMESPACE>`
 
 Locate and enter the following in Settings: 
 
```

Comment:
Should this be `kubectl get service output-processor --namespace=<YOUR_CIRCLECI_NAMESPACE>`

Reply:
@circleci/server

Reply:
@ryanwohara just checking on these two comments if the change I suggest would be correct?

Reply:
Agreed it should not just be circleci-server - I personally use rohara-dev!

Reply:
Thanks!